atlp-rwanda/Dev-Pulse-BN
# Ft export trainee data

> #### What does this PR do?
> > export data
> 
> #### Description of Task to be completed?
> * export trainee's data
> * providing start and end date

> #### How should this be manually tested?
> 1. clone this repo
> 2. Make sure you are using the latest version on node and
> 3. create a `.env` file and copy all variables from `.env.example`
> 4. run `yarn install`
> 5. go to `http://localhost:3000/api/v1/user/:id/ratings`
> houZrGON
> #### Any background context you want to provide?
> N/A
> 
> #### What are the relevant pivotal tracker stories?
> 
> #### Screenshots (if appropriate)
> N/A
> 
> #### Questions:
> N/A
